### ISSUE TITLE
Default learning rate changed in the 1.0-alp release

### ISSUE THREAD

**Issue by wanjizheng** [deeplearning4j/deeplearning4j#5040]
In the 0.9.1 version, the default learning rate value is 0.1 if you don't give it an initial value.  This behaviour changed in the 1.0-alp release.  Therefore, the result generated from the neural network could be very different from the 0.9.1 version if you don't have an initial learning rate value.

I'm not sure what is the new default value, but please could you update the release notes to highlight this change (https://deeplearning4j.org/releasenotes) or change the new default value to 0.1 again?

Thanks a lot!

**Comment by AlexDBlack** [comment]
Thanks for the issue - but I think it's a bad idea to mention a "default" learning rate in the docs. It's different for different updaters (Adam, RMSProp, etc) and some updaters don't even have a learning rate (AdaDelta, NoOp) (you can see that in the javadoc).

The fact that there even was a default learning rate was one reason for the API changes - every neural network needs the learning rate to be tuned, hence the concept of a default isn't helpful or useful (i.e., the "default" isn't appropriate or optimal >95% of the time). I don't want to talk about "default" learning rates in the docs when the default isn't something people should be using anyway.
